website/source/docs/commands/job-dispatch.html.md.erb

```diff
@@ -0,0 +1,111 @@
+---
+layout: "docs"
+page_title: "Commands: job dispatch"
+sidebar_current: "docs-commands-job-dispatch"
+description: >
+  The dispatch command is used to create an instance of a parameterized job.
+---
+
+# Command: job dispatch
+
+~> The `job dispatch` subcommand described here is available only in version
+0.5.3 and later. The release canidate is downloadable on the [releases
+page.](https://releases.hashicorp.com/nomad/0.5.3-rc1/)
+
+The `job dispatch` command is used to create new instances of a [parameterized
```

Comment:
You are on the command page and it is kinda weird but that is the syntax for linking. [parameterized job] there links to the correct job-spec page.